### DISCUSSION TITLE
Enums with x-enumDescriptions occupy too much visual space

### DISCUSSION THREAD

**Discussion by federicobond** [scalar/scalar#3155] [Ideas]
When enums have descriptions set with `x-enumDescriptions`, the rendering switches from a bulleted list to the property list display depicted below. When you have more than a few options this arrangement interrupts the reading flow and gives too much visual weight to the enum values compared to the rest of the properties, especially when there are many items.

<img width="462" alt="Captura de pantalla 2024-09-20 a la(s) 12 00 29" src="https://github.com/user-attachments/assets/48195ea6-9ebb-4c66-a1f9-b05c8938c861">

Would it make more sense to render enum descriptions like this?

 * one (One)
 * two (Two)

Or:

 * `one` - One
 * `two` - Two

**Comment by hanspagel** [comment]
@cameronrohani Wdyt?

**Comment by federicobond** [comment]
Hi! Just following up on this. Would you be open to a contribution either way?
